emission/analysis/classification/inference/mode.py

```diff
@@ -270,42 +261,41 @@ def getGroundTruthMode(self, section):
 # 19. both start and end close to bus stop
 # 20. both start and end close to train station
 # 21. both start and end close to airport
-  def updateFeatureMatrixRowWithSection(self, featureMatrix, i, section):
-    if 'distance' in section.keys():
-      featureMatrix[i,0] = section['distance']
-    else:
-      featureMatrix[i, 0] = easf.calDistance(section['start_loc']['coordinates'], section['end_loc']['coordinates'])  #####would need to calculate this
-
-    featureMatrix[i, 1] = (section['end_ts'] - section['start_ts'])
-
+  def updateFeatureMatrixRowWithSection(self, featureMatrix, i, section): 
+    
+    featureMatrix[i,0] = section['distance']
+    
+    
+    featureMatrix[i, 1] = section['duration']
+  
     # Deal with unknown modes like "airplane"
     try:
       featureMatrix[i, 2] = section['sensed_mode']
     except ValueError:
       featureMatrix[i, 2] = 0
 
-    featureMatrix[i, 3] = section['section_id']   ####
-    #featureMatrix[i, 4] = easf.calAvgSpeed(section)
+    #featureMatrix[i, 3] = section['_id']   ####
```

Comment:
Yes, I intentionally commented out the `section[_id]` bit because it is a string and not a number, so it makes so sense to put it as an entry in the matrix.